variants/README.md

```diff
@@ -336,6 +336,16 @@ This variant includes the (optional) `image-layout` section, which allows the us
 Though we've done so here for sake of demonstration, resizing the "data" disk image isn't necessary as it expands to fill the disk on boot.
 `partition-plan` is the strategy used for image partitioning, with the options being "split" (the default) or "unified".
 The "split" partition strategy has separate volumes for "OS" and "data", while "unified" has "OS" and "data" on a single volume.
+
+All `aws` variants use the "split" partition plan and the published AMIs all have two EBS volumes.
+The root volume contains the "OS" image and the data volume contains the "data" image.
```

Comment:
This documentation note makes sense but I wonder if it'll be read by people who need to see it. It's also a bit out of place since it discusses AWS specifics in the general variant feature doc.

It might be better to write this up as an announcement in the discussions area, as we did for the SELinux related changes.